.github/workflows/code_format.yml

```diff
@@ -47,7 +47,7 @@ jobs:
                1. Run `git apply clang-format-diff.patch` to make the changes
                1. Commit and push up the formatted code
             1. Run `clang-format` locally
-               1. Run the command `bash -c 'shopt -s nullglob; for i in ./{.,src,src/inc,boards/*}/*.{c,cpp,h,hpp}; do clang-format -i $i; done'`
+               1. Run the command `bash -c 'shopt -s nullglob globstar;GLOBIGNORE=./src/libs/TimerInterrupt/*; for i in ./{.,src/**,unit_tests,boards/**}/*.{c,cpp,h,hpp}; do clang-format -i $i; done'`
```

Comment:
I'm going to make a future PR to make a custom `pio code-format` command that will handle this in a python script, because this is just getting unwieldy